frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -333,10 +296,15 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType<P
                 )
                 if (currentPlayerTime !== null) {
                     const nextPlayerTime = currentPlayerTime + values.jumpTimeMs
```

Comment:
I think this can be similarly clamped with a Math.min

Reply:
I think this is tied in with the behavior above. Agreed that we want a clamp here if we don't want the "reset" behavior.